src/main/scala/BigNumbers.scala

```diff
@@ -0,0 +1,43 @@
+class BigNumbers {
+
+  def bigNumberConverter(num:BigInt) = {
+    val number = num.toString
+    val startNumber = number.substring(0,number.length%3)
+    var threesList = scala.collection.mutable.ListBuffer[String]()
+    if(startNumber != "") threesList += startNumber
+    val newNumber = number.substring(number.length%3,number.length)
+    val numArray = newNumber.toCharArray
+    for(i <- 0 to numArray.length-3 by 3) {
+      threesList += numArray(i).toString + numArray(i + 1).toString + numArray(i + 2).toString
+    }
+    threesList.reverse.toList
+  }
+
+  def makeSomeWords(bigNumberList: List[String]): String = {
+    var shortScaleString = ""
+    var longScaleString = ""
+    for(i <- bigNumberList.length-1 to 0 by -1) {
```

Comment:
pretty sure could do this recursively and get rid of the var shortScaleString = "", var longScaleString = ""